en/simulation/gazebo.md

```diff
@@ -236,90 +255,72 @@ This thin wrapper simply embeds app arguments like current working directories a
 To start Gazebo and PX4 separately:
 
 * Run gazebo (or any other sim) server and client viewers via the terminal:
-  ```
+  ```sh
```

Comment:
@julianoes The command on next line no longer works.

What is the right way to start SITL for use with an IDE? I know on VSCode that it is not needed, but presumably still for Eclipse? Who would know?

Reply:
I have no idea, I never really understood this. I would remove it for now. If people that require it want it again they can contribute that again.

What you can do though is `make px4_sitl none` which might serve the same purpose.

Reply:
@julianoes I'm not sure. What does `_ide` actually *do*? Note there are a bunch of ide options still present - e.g. `gazebo_iris_ide` so not sure.

Do you know anyone who debugs using these or who might know?

Reply:
I just checked, this actually still works. You just need to add these extra underscores.

Reply:
@julianoes Can you be explicit? ie ` make px4_sitl gazebo_none_ide` changes to what?

Reply:
These two variants work:
```
make px4_sitl gazebo___ide
make px4_sitl gazebo_iris_ide
```

Reply:
Thanks very much. Integrated and will now merge.